src/Microsoft.SqlTools.SqlCore/ObjectExplorer/StatelessObjectExplorer.cs

```diff
@@ -0,0 +1,98 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.Data.SqlClient;
+using Microsoft.SqlServer.Management.Common;
+using Microsoft.SqlTools.SqlCore.Connection;
+using Microsoft.SqlTools.SqlCore.ObjectExplorer;
+using Microsoft.SqlTools.SqlCore.ObjectExplorer.Nodes;
+using Microsoft.SqlTools.SqlCore.ObjectExplorer.SmoModel;
+
+namespace Microsoft.SqlTools.CoreSql.ObjectExplorer
+{
+    public class StatelessObjectExplorer
```

Comment:
Please add xml doc explaining what this class is for and how it differs from the "normal" OE